app/src/main/java/com/concordia/mcga/utilities/pathfinding/PathFinder.java

```diff
@@ -0,0 +1,102 @@
+package com.concordia.mcga.utilities.pathfinding;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Pathfinding class, which runs the A* shortest pathfinding algorithm
+ */
+public class PathFinder {
+
+    TiledMap map;
+    Set<PathFinderTile> openSet;
+    Set<PathFinderTile> closedSet;
+
+    public PathFinder(TiledMap map) {
+        openSet = new HashSet<>();
+        closedSet = new HashSet<>();
+        this.map = map;
+    }
+
+    /**
+     *  Finds the shortest path between the given x,y coordinates
+     * @param startX - starting position's x coordinate
+     * @param startY - starting position's y coordinate
+     * @param destX - ending position's x coordinate
+     * @param destY - ending position's y coordinate
+     * @return - Returns a list of the tiles found in the shortest path. Sorted from first to last.
+     * @throws Exception - Thrown if there exists no valid path between both points
+     */
+    public List<PathFinderTile> shortestPath(int startX, int startY, int destX, int destY)
```

Comment:
I think it better to create a Point class that has x and y. it will make the code easier to read.
Then your method would become:
public List<PathFinderTile> shortestPath(Point start, Point Destination)